recipe/meta.yaml

```
@@ -1,22 +1,31 @@
 
 {% set name = "STIR" %}
-{% set version = "6.1.0" %}
-{% set build_number = 3 %}
+{% set version = "6.2.0" %}
+{% set build_number = 0 %}
 {% if cuda_compiler_version != "None" %}
 {% set build_number = build_number + 200 %}
 {% endif %}
 
+{% if cuda_compiler_version in (None, "None", True, False) %}
+{% set cuda_major = 0 %}
+{% else %}
+{% set cuda_major = environ.get("cuda_compiler_version", "11.8").split(".")[0] | int %}
+{% endif %}
+
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/UCL/STIR/archive/refs/tags/rel_{{ version }}.tar.gz
-  sha256: 638121f9b3c21e88cc1b6e6f66d8a89ed2902f9a84a546dfe0b373ea9aa5ce11
+  sha256: c2c072b50545a16484951a6412ab5bbb2944320f5c7de12ead7329725d3d7044
 
 build:
   number: {{ build_number }}
-  skip: true  # [linux and cuda_compiler_version == "10.2"]
+  # skipping CUDA 11 builds on Windows due to missing cudart.dll problems, see https://github.com/conda-forge/stir-feedstock/pull/121
+  skip: true  # [win and (cuda_major<12)]
```

This probably doesn't work out with the different parsing phases (jinja vs. selectors etc.).
```suggestion
  {% if cuda_major < 12 %}
  skip: true  # [win]
  {% endif %}
```